fern/definition/admin/cluster/__package__.yml

```diff
@@ -5,7 +5,7 @@ imports:
 
 service:
```

Comment:
needs dc-related endpoints

Reply:
Not sure I know what you mean, could you elaborate?